elements/pl_checkbox/pl_checkbox.py

```diff
@@ -102,11 +106,29 @@ def render(element_html, element_index, data):
                 + ' />\n' \
                 + '    (' + answer['key'] + ') ' + answer['html'].strip() + '\n' \
                 + '  </label>\n'
+            if answer['key'] in submitted_keys:
+                if answer['key'] in correct_keys:
+                    item = item + '<span class="label label-success"><i class="fa fa-check" aria-hidden="true"></i></span>&nbsp&nbsp&nbsp&nbsp'
+                else:
+                    item = item + '<span class="label label-danger"><i class="fa fa-times" aria-hidden="true"></i></span>&nbsp&nbsp&nbsp&nbsp'
             if not inline:
                 item = '<div class="checkbox">\n' + item + '</div>\n'
             html += item
         if inline:
             html = '<p>\n' + html + '</p>\n'
+        partial_score = data['partial_scores'].get(name, {'score': None})
+        score = partial_score.get('score', None)
+        if score is not None:
+            try:
+                score = float(score)
+                if score >= 1:
+                    html = html + '&nbsp<span class="label label-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>'
+                elif score > 0:
+                    html = html + '&nbsp<span class="label label-warning"><i class="fa fa-circle-o" aria-hidden="true"></i> {:d}%</span>'.format(math.floor(score * 100))
+                else:
+                    html = html + '&nbsp<span class="label label-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>'
+            except:
+                raise ValueError('invalid score' + score)
```

Comment:
The `submission` panel render doesn't include the score.

Reply:
I don't know how I missed this. Fixed.